src/tests/arbitrum/btc3x.test.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable  @typescript-eslint/no-non-null-assertion */
+import { IndexCoopBitcoin3xIndex } from 'constants/tokens'
+import {
+  getSignerAccount,
+  LocalhostProviderArbitrum,
+  QuoteTokens,
+  TestFactory,
+  wei,
+  ZeroExApiArbitrumSwapQuote,
+} from 'tests/utils'
+
+const { eth } = QuoteTokens
+const eth3x = {
+  address: IndexCoopBitcoin3xIndex.addressArbitrum!,
+  decimals: 18,
+  symbol: IndexCoopBitcoin3xIndex.symbol,
+}
+
+describe.skip('BTC3X (Arbitrum)', () => {
```

Comment:
looking to keep this test suite skipped?

Reply:
Yes for now. As mentioned might have to skip the whole arbitrum tests suite to release the SDK due to the bug causing failed tx's.